packages/cli/src/config/webpack.config.common.js

```diff
@@ -114,6 +114,20 @@ module.exports = ({ config, context }) => {
     },
 
     plugins: [
+      new HtmlWebpackPlugin({
+        filename: path.join(context.publicDir, context.indexPageTemplate),
+        template: path.join(context.scratchDir, context.indexPageTemplate),
+        chunksSortMode: 'dependency',
+        ...customOptions
```

Comment:
Would like to look into something like #199 so that for plugins like Analytics, it could be disabled for development, and only enabled for production builds.